libfwupd/fwupd-common.h

```diff
@@ -10,10 +10,35 @@
 
 G_BEGIN_DECLS
 
+/**
+ * FWUPD_DBUS_PATH:
+ *
+ * The dbus path
+ *
+ **/
 #define FWUPD_DBUS_PATH			"/"
+/**
+ * FWUPD_DBUS_SERVICE:
+ *
+ * The dbus service
+ *
+ **/
 #define FWUPD_DBUS_SERVICE		"org.freedesktop.fwupd"
+/**
+ * FWUPD_DBUS_INTERFACE:
+ *
+ * The dbus interface
+ *
+ **/
+
 #define FWUPD_DBUS_INTERFACE		"org.freedesktop.fwupd"
```

Comment:
I think we should probably take this opportunity to expand out all these -- making them at least into single sentences (e.g. with a full stop at the end) -- for instance, `The main D-Bus interface which is exported by the fwupd daemon which clients can call methods on and register for signals.` I know it's a ton of work, and I'm happy to help with this when I get back.